docs/source/managed-federation/uplink.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Apollo Uplink
+sidebar_title: Uplink
+---
+
+When using [managed federation](./overview/), your federated gateway regularly polls an endpoint called **Apollo Uplink** for its latest supergraph schema and other configuration:
+
+```mermaid
+graph LR;
+  subgraph "Your infrastructure"
+  serviceA[Products subgraph];
+  serviceB[Reviews subgraph];
+  gateway([Gateway]);
+  end
+  subgraph "Apollo cloud"
+  registry{{Apollo Schema Registry}};
+  uplink{{Apollo Uplink}}
+  end
+  serviceA & serviceB -->|Pushes schema| registry;
+  registry -->|Updates config| uplink;
+  gateway -->|Polls for config changes| uplink;
+  class registry secondary;
+  class uplink secondary;
+```
+
+To maximize uptime, Uplink is hosted simultaneously at _two_ endpoints, one in GCP and one in AWS:
+
+* GCP: `https://uplink.api.apollographql.com/`
+* AWS: `https://aws.uplink.api.apollographql.com/`
+
+By default in versions of `@apollo/gateway` vTODO and later, your gateway polls the GCP URL at a regular interval. Whenever a poll request fails, the gateway retries the request with the AWS URL. It continues swapping between the two URLs until the request succeeds, or until reaching the maximum number of retries.
```

Comment:
minor: perhaps we should say the "default" URL instead of the GCP URL (just whatever is the first in the list), to be more general? I'm not sure how likely users are to override the endpoints, but that would give them the understanding of how the behavior works (it just so happens that GCP is first in the list by default)

Reply:
ah, nvm! You cover this very well directly below. Thank you 🙏 I should've kept reading before commenting!